packages/@aws-cdk/aws-events/lib/rule.ts

```diff
@@ -178,32 +179,30 @@ export class Rule extends Resource implements IRule {
       const sourceAccount = sourceStack.account;
       const sourceRegion = sourceStack.region;
 
-      if (targetRegion !== sourceRegion) {
-        throw new Error('Rule and target must be in the same region');
-      }
-
-      if (targetAccount !== sourceAccount) {
-        // cross-account event - strap in, this works differently than regular events!
+      if (targetAccount !== sourceAccount || targetRegion !== sourceRegion) {
+        // cross-account and/or cross-region event - strap in, this works differently than regular events!
         // based on:
-        // https://docs.aws.amazon.com/eventbridge/latest/userguide/eventbridge-cross-account-event-delivery.html
+        // https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-cross-account.html
 
-        // for cross-account events, we require concrete accounts
+        // for cross-account or cross-region events, we require concrete accounts
         if (Token.isUnresolved(targetAccount)) {
-          throw new Error('You need to provide a concrete account for the target stack when using cross-account events');
+          throw new Error('You need to provide a concrete account for the source stack when using cross-account or cross-region events');
         }
         if (Token.isUnresolved(sourceAccount)) {
-          throw new Error('You need to provide a concrete account for the source stack when using cross-account events');
+          throw new Error('You need to provide a concrete account for the source stack when using cross-account or cross-region events');
         }
         // and the target region has to be concrete as well
         if (Token.isUnresolved(targetRegion)) {
-          throw new Error('You need to provide a concrete region for the target stack when using cross-account events');
+          throw new Error('You need to provide a concrete region for the target stack when using cross-account or cross-region events');
         }
 
         // the _actual_ target is just the event bus of the target's account
-        // make sure we only add it once per account
-        const exists = this.accountEventBusTargets[targetAccount];
-        if (!exists) {
+        // make sure we only add it once per account per region
```

Comment:
Shouldn't we check for a pair of `account,region` then, instead of different sets for account and regions?

Reply:
Hi, I really need this field, any plan to merge?
I see this is idle since many days, If needed I'm available to work on this

Reply:
@gianlucb I would really appreciate some help on this if you're able to? This was my first pull request and struggling a bit.

Reply:
> @gianlucb I would really appreciate some help on this if you're able to? This was my first pull request and struggling a bit.

I have addressed the comments from **rix0rrr**. Can you give me write permissions to your fork? I will push my changes, so we can merge this PR

Reply:
@gianlucb 👍